pkg/capture/flow.go

```diff
@@ -223,30 +223,31 @@ func (f *FlowLog) Rotate() *hashmap.AggFlowMap {
 }
 
 func (f *FlowLog) transferAndAggregate() (agg *hashmap.AggFlowMap) {
+
+	// Initialize aggregate flow map / result
 	agg = hashmap.NewAggFlowMap()
 
-	// Reusable key conversion buffers
+	// Create reusable key conversion buffers
 	keyBufV4, keyBufV6 := types.NewEmptyV4Key(), types.NewEmptyV6Key()
-	var goDBKey types.Key
 
 	for k, v := range f.flowMap {
 
-		// Populate key buffer according to source flow
```

Comment:
This was moved inside the condition below (because it is not required for flows that get dropped).